tools/cloud-build/daily-tests/ansible_playbooks/slurm-integration-test.yml

```diff
@@ -267,6 +267,15 @@
       ansible.builtin.debug:
         var: suspend_output.stdout_lines
 
+    - name: Check for stockout errors
```

Comment:
Please move the check for stockout errors to a separate PR.

Reply:
Done